docs/controller/ingress/example.yml

```diff
@@ -16,9 +16,9 @@
 # Verify that the route has been added in the simulator/lb1:
 # infrakit simulator/lb1 routes ls
 #
-# FRONTEND PORT    PROTOCOL   BACKEND PORT  CERT
-# 7777             TCP        7777  <-- from the swarm service
-# 80               http       8080  <-- from the static route in the config
+# FRONTEND PORT    FRONTEND PROTOCOL      BACKEND PORT      BACKEND PROTOCOL        CERT
```

Comment:
Why are these changes here? Is this file out of date? Can you put something in the commit message explaining why this is updated? It seems like that the `Protocol` case is the main delta here (`http` -> `HTTP`).

Reply:
They've simply been sitting in my branch waiting to be put in as David said to put them in with my next PR. I could certainly push them in another PR if that is warranted.

Reply:
Let's remove changes to this file in this PR...

Reply:
Removed.